LayoutTests/platform/glib/imported/w3c/web-platform-tests/cookies/attributes/max-age-expected.txt

```diff
@@ -4,9 +4,9 @@ PASS Set cookie with age
 PASS Set no cookie with max-age=0
 PASS Set no cookie with max-age=-1
 PASS Set no cookie with max-age=-20
-PASS Set multiple cookies with max-age attribute
+FAIL Set multiple cookies with max-age attribute assert_equals: The cookie was set as expected. expected "testA=6; testB=6" but got "testB=6; testA=6"
 PASS Expire later cookie with same name and max-age=0
 PASS Expire later cookie with same name and max-age=0, and don't set cookie with max-age=0
-PASS Set mulitiple cookies with valid max-age values
+FAIL Set mulitiple cookies with valid max-age values assert_equals: The cookie was set as expected. expected "test=\"9! = foo; test9" but got "test9; test=\"9! = foo"
```

Comment:
This reordering bug seems to be affecting many of the tests.

Reply:
Fun fact, `soup_cookie_jar_all_cookies()` is always in reverse order.